moduleroot/.travis.yml

```diff
@@ -37,5 +37,8 @@ matrix:
     # Only Puppet 4.x supports Ruby 2.2. Also limit the OS set we test Ruby 2.2 with.
     - rvm: 2.2.3
       env: PUPPET_VERSION=4.0 ONLY_OS="debian-8-x86_64,centos-7-x86_64,ubuntu-16-x86_64,freebsd-10-amd64,windows-2012 R2-x64"
+    # Only Puppet >= 4.4 supports Ruby 2.3. Also limit the OS set we test Ruby 2.3 with.
+    - rvm: 2.3.0
+      env: PUPPET_VERSION=4.0 ONLY_OS="debian-8-x86_64,centos-7-x86_64,ubuntu-16-x86_64,freebsd-10-amd64,windows-2012 R2-x64"
```

Comment:
I'd like to run that test with every major OS. As newer Rubies are faster, this is also quite cheap.